src/transformers/models/bert/modeling_bert.py

```diff
@@ -1083,7 +1084,7 @@ def forward(
 
         total_loss = None
         if labels is not None and next_sentence_label is not None:
-            loss_fct = CrossEntropyLoss()
+            loss_fct = F.cross_entropy
```

Comment:
Probably should consistently drop all `loss_fct` vars and use `F.cross_entropy` everywhere else in this file? You already did most of them, just a few left.

Reply:
You're right!
Changed this